CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.90.0
+
+### Breaking changes
+
+* @PrimaryKey field value can be null for String, Byte, Short, Integer, and Long types, and should be migrated by using RealmObjectSchema.setNullable() for older version of Realm (#2515).
```

Comment:
I would rephrase this to: 

@PrimaryKey field value can be null for String, Byte, Short, Integer, and Long types`. Older Realms should be migrated using RealmObjectSchema.setNullable() (#2515).